src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -1,799 +1,3 @@
-# https://github.com/mono/mono/issues/15160
--nomethod System.Threading.Tests.SemaphoreTests.OpenExisting_NotSupported_Unix
-
-# Expects PlatformNotSupportedException but none thrown
-# https://github.com/mono/mono/issues/15161
--nomethod System.Threading.Tests.SemaphoreTests.Ctor_NamesArentSupported_Unix
-
-####################################################################
-##  System.Threading.Tests
-####################################################################
-
-# Requires precise GC (should be ignored in dotnet/corefx for mono)
--nomethod System.Threading.Tasks.Tests.ExecutionContextFlowTest.TaskCompletionSourceDoesntCaptureExecutionContext
-
-# flaky, causes stack overflow
--nomethod System.Threading.Tasks.Tests.TaskContinueWithTests.LongContinuationChain_ContinueWith_DoesNotStackOverflow
-
-####################################################################
-##  System.Threading.ThreadPool.Tests
-####################################################################
-
-# Tests for ThreadPool.SetMaxThreads(-1, -1) == true, which only happens to work on CoreCLR because
-# the managed SetMaxThreadsNative prototype uses "int"s while the unmanaged code uses "DWORD"s and
-# thus it interprets it as large positive numbers.
-#
-# https://github.com/mono/mono/issues/15164
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinMaxThreadsTest
-
-# TODO: Differences in behaviour between NetFX and CoreFX
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinThreadsTo0Test
-
-# Explicitly skipped if it's MonoVM (SkipOnTargetFramework)
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinMaxThreadsTest_ChangedInDotNetCore
-
-####################################################################
-##  System.ComponentModel.Composition.Tests
-####################################################################
-
--nomethod System.ComponentModel.Composition.CompositionServicesTests.GetDefaultContractNameTest
--nomethod System.ComponentModel.Composition.MetadataViewProviderTests.GetMetadataView_InterfaceWithIndexer_ShouldThrowNotSupportedException
--nomethod Tests.Integration.ExportFactoryTests.ExportFactoryStandardImports_ShouldWorkProperly
-
-# flaky test - System.TypeLoadException : Could not load type '_proxy_System.ComponentModel.Composition.IStronglyTypedStructure_3b7f133d-79c7-449e-8f68-22aa093dbb8d' from assembly ''.
--nomethod System.ComponentModel.Composition.MetadataAttributeTests.StronglyTypedStructureTest
--nomethod System.ComponentModel.Composition.MetadataAttributeTests.StronglyTypedStructureTestWithTransparentViews
--nomethod System.ComponentModel.Composition.MetadataViewProviderTests.GetMetadataView_IMetadataViewWithDefaultedInt64
--nomethod System.ComponentModel.Composition.ExportCollectionTests.ImportCollectionsFromContainerOnly
-# disable the whole namespace, https://github.com/mono/mono/issues/16417
--nonamespace System.ComponentModel.Composition
```

Comment:
It looks like you didn't actually disable this whole namespace (we had more tests fail there in the latest build): 

https://github.com/MaximLipnin/runtime/blob/4d89064eeef28e0f57eb5df78c3a8e371851a52a/src/libraries/System.ComponentModel.Composition/tests/System/ComponentModel/Composition/CompositionContainerTests.cs#L14-L17

Reply:
Disabled